src/main/resources/i18n/messages.properties

```diff
@@ -33,3 +33,8 @@ oauth2.scope.administrative_actions=Perform administrative actions
 oauth2.scope.administrative_actions.description=Such as kick, ban, broadcast message, ...
 oauth2.scope.read_sensible_userdata=View sensible user data
 oauth2.scope.read_sensible_userdata.description=Such as email, steam id, bans, ...
+ban.title=You are Banned from FAForever
+ban.expiration=Ban expires at:
+ban.permanent=Permanently Banned
```

Comment:
On permaban I'd write it in a manner such as `Ban expires at: never`

Reply:
You got it